### PR TITLE
ON-10 CRUD order

### PR DESCRIPTION
**Pull request by dinhlcph18273**

### CODE REVIEW

File: src/api/v1/controllers/order.controller.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const orderService = require('../services/order.service');
+const catchAsync = require('../utils/catchAsync');
+
+module.exports = {
+  getAllOrder: catchAsync(async (req, res, next) => {
+    const order = await orderService.findAll();
```
Comment (vanchuongkieu) [inline src/api/v1/controllers/order.controller.js:6]: order => orders

File: src/api/v1/services/order.service.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const _order = require('../models/order.model');
+
+module.exports = {
+  findAll: async () => {
+    const orders = await _order.find({});
+    return orders;
+  },
+  findOne: async (_id) => {
+    const orders = await _order.findOne(_id);
```
Comment (vanchuongkieu) [inline src/api/v1/services/order.service.js:9]: orders => order

File: src/api/v1/services/order.service.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    return orders;
+  },
+  createOrder: async (data) => {
+    const orders = await new _order({ ...data }).save();
```
Comment (vanchuongkieu) [inline src/api/v1/services/order.service.js:13]: orders => order

File: src/api/v1/services/order.service.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    return orders;
+  },
+  updateOrder: async (_id, data) => {
+    const orders = await _order.findOneAndUpdate(_id, data, { new: true }).exec();
```
Comment (vanchuongkieu) [inline src/api/v1/services/order.service.js:17]: orders => order

File: src/api/v1/services/order.service.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    return orders;
+  },
+  deleleOrder: async (_id) => {
+    const orders = await _order.findOneAndDelete(_id).exec();
```
Comment (vanchuongkieu) [inline src/api/v1/services/order.service.js:21]: orders => order

**Review by vanchuongkieu — APPROVED** [review]

**Review by HiamQuan — APPROVED** [review]

**Pull request merged**